event.go

```diff
@@ -23,4 +23,6 @@ type ScrollEvent struct {
 type DragEvent struct {
 	PointEvent
 	DraggedX, DraggedY int
+	DraggedTotal       Position
```

Comment:
Exposing this as a Position feels wrong, as it is a distance.
It got me thinking - is this needed? and if it is then we should match the DraggedX and DraggedY format above.
Anyone listening to this event would be able to calculate it by summing the X and Y no?

Reply:
I agree regarding the presentation format. It would be cool to have a `fyne.Distance` for this because it reduces the amount of fields of the `DragEvent`. But since the dragged instance already is a pair of `int` I'll probably do it this way.

I disagree on the cumulation of the dragged delta values.
First, the listeners don't have a start/end indicator.
Second, the listener should not have to monitor the state of dragging.